ruby_event_store-browser/devserver/.gitignore

```diff
@@ -0,0 +1,3 @@
+public/ruby_event_store_browser.css
+public/ruby_event_store_browser.js
+public/ruby_event_store_browser.map
```

Comment:
So far we've kept everything in top-level repo gitignore. Does gitignore work recursively?

Reply:
It does, but I can move it :+1: